__tests__/app.test.js

```diff
@@ -92,7 +92,36 @@ describe('Backend testing', () => {
         })
     });
 });
+describe.only('GET /api/users', () => {
+    test('status:200 - returns all users within data', () => {
+        return request(app)
+        .get('/api/users')
+        .expect(200)
+        .then(({body : {users}})=>{
+            expect(users).toBeInstanceOf(Array)
+            users.forEach(user => {
+                expect(user).toEqual(
+                expect.objectContaining({
+                    username: expect.any(String),
+                    name: expect.any(String),
+                    avatar_url: expect.any(String),
```

Comment:
test looks good! could be even more secure by testing for the length of the array you get back, making sure it's the same length as you expect